db/migrate/20241230044238_add_include_in_kind_values_in_exported_files_to_organizations.rb

```diff
@@ -0,0 +1,6 @@
+class AddIncludeInKindValuesInExportedFilesToOrganizations < ActiveRecord::Migration[7.2]
+  def change
+    add_column :organizations, :include_in_kind_values_in_exported_files, :boolean
```

Comment:
This should be NOT NULL. Otherwise it's got three values, true false and null. :)